docs/pdoc_templates/menu.mako

```diff
@@ -1,142 +1,494 @@
 <!-- ------- -->
+<link rel="stylesheet" href="https://label-studio-docs-new-theme.netlify.app/css/styles.css">
```

Comment:
`https://label-studio-docs-new-theme.netlify.app` is just for review purposes. I’ll remove it before we merge this.